py/asmrv32.h

```diff
@@ -735,8 +753,8 @@ static inline uint8_t asm_rv32_allowed_extensions(void) {
 #define REG_TEMP2 ASM_RV32_REG_T3
 #define REG_FUN_TABLE ASM_RV32_REG_S1
 #define REG_LOCAL_1 ASM_RV32_REG_S3
-#define REG_LOCAL_2 ASM_RV32_REG_S4
-#define REG_LOCAL_3 ASM_RV32_REG_S5
+#define REG_LOCAL_2 ASM_RV32_REG_S2
+#define REG_LOCAL_3 ASM_RV32_REG_S4
```

Comment:
Why did these change?  And why not make them sequential, LOCAL_1 through LOCAL_3 mapping to S2 through S4?